gammapy/modeling/models/tests/test_temporal.py

```diff
@@ -288,3 +295,21 @@ def test_plot_constant_model():
     constant_model = ConstantTemporalModel(const=1)
     with mpl_plot_check():
         constant_model.plot(time_range)
+
+
+def test_io():
```

Comment:
Minor: maybe move to `gammapy/modeling/models/test_io.py` instead? This where the other I/O tests are...

Reply:
done